core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -20,9 +20,9 @@
 oppia.factory('explorationData', [
   '$http', '$log', 'alertsService',
   'EditableExplorationBackendApiService',
-  'ReadOnlyExplorationBackendApiService', '$q',
+  'ReadOnlyExplorationBackendApiService','$q', 'UrlInterpolationService',
```

Comment:
Why import ``UrlInterpolationService``? It isn't being used in this service from what I can tell.